edge/pkg/clusterd/edgecluster_status_reporter.go

```diff
@@ -158,23 +153,34 @@ func (esr *EdgeClusterStatusReporter) getEdgeClusterStatusRequest(edgeCluster *e
 	var edgeClusterStatus = &edgeapi.EdgeClusterStatusRequest{}
 	edgeClusterStatus.UID = esr.clusterd.uid
 	edgeClusterStatus.Status = *edgeCluster.Status.DeepCopy()
-	edgeClusterStatus.Status.Healthy = helper.TestClusterReady()
-
-	edgeClusterStatus.Status.EdgeClusters = helper.GetLocalClusterScopeResourceNames("edgeclusters", "")
-	edgeClusterStatus.Status.Nodes = helper.GetLocalClusterScopeResourceNames("nodes", "")
-	edgeClusterStatus.Status.EdgeNodes = helper.GetLocalClusterScopeResourceNames("nodes", "node-role.kubernetes.io/edge")
-
-	var receivedMissions []string
-	var matchededMissions []string
-	for k, v := range esr.missionDeployer.MissionMatch {
-		receivedMissions = append(receivedMissions, k)
-		if v {
-			matchededMissions = append(matchededMissions, k)
+
+	clusterHealthy := helper.TestClusterReady()
+
+	if clusterHealthy {
+		edgeClusterStatus.Status.Healthy = "healthy"
+		edgeClusterStatus.Status.EdgeClusters = helper.GetLocalClusterScopeResourceNames("edgeclusters", "")
+		edgeClusterStatus.Status.Nodes = helper.GetLocalClusterScopeResourceNames("nodes", "")
+		edgeClusterStatus.Status.EdgeNodes = helper.GetLocalClusterScopeResourceNames("nodes", "node-role.kubernetes.io/edge")
+
+		var receivedMissions []string
+		var matchededMissions []string
+		for k, v := range esr.missionDeployer.MissionMatch {
+			receivedMissions = append(receivedMissions, k)
+			if v {
+				matchededMissions = append(matchededMissions, k)
+			}
 		}
-	}
 
-	edgeClusterStatus.Status.ReceivedMissions = receivedMissions
-	edgeClusterStatus.Status.ActiveMissions = matchededMissions
+		edgeClusterStatus.Status.ReceivedMissions = receivedMissions
+		edgeClusterStatus.Status.ActiveMissions = matchededMissions
+	} else {
+		edgeClusterStatus.Status.Healthy = "unreachable"
```

Comment:
same here. "unreachable" in a constant variable.

Reply:
done